build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -40,24 +40,13 @@ metadata:
           },
           "spec": {
             "version": "v2.6",
-            "tracing": {
-              "type": "Jaeger",
-              "sampling": 10000
-            },
             "policy": {
                "type": "Istiod"
             },
             "telemetry": {
                "type": "Istiod"
             },
             "addons": {
-              "jaeger": {
-                "install": {
-                  "storage": {
-                    "type": "__JAEGER_STORAGE__"
```

Comment:
You can delete all uses of `JAEGER_STORAGE` and `__JAEGER_STORAGE__` in `build/generate-manifests.sh` too.